### ISSUE TITLE
[Bug]: Very slow startup with Svelte/Vite using SCSS

### ISSUE THREAD

**Issue by Sahasrara** [carbon-design-system/carbon#15373]
### Package

carbon-components

### Browser

Edge

### Package version

10.58.12

### React version

N/A

### Description

I've been trying to use the SCSS version of the carbon styling based off the suggestion from [this documentation](https://svelte.carbondesignsystem.com/) that it would be more efficient. 

> The most performant method to load styles is to import SCSS directly from carbon-components.

I installed carbon components, imported it with:
`@import 'carbon-components/scss/globals/scss/styles.scss';`

And found that the time it took to load my app went up ~30x.

### Reproduction/example

I don't have one right now.

### Steps to reproduce

I'm using Tauri:
https://tauri.app/v1/guides/getting-started/setup/sveltekit

And then:
https://svelte.carbondesignsystem.com/

And finally:
https://github.com/carbon-design-system/carbon/blob/v10/docs/guides/sass.md

Then I tried to import:
`@import 'carbon-components/scss/globals/scss/styles.scss';`

And it the empty app takes a LONG time to become responsive.

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Yes, the sass builds for `carbon-components@v10` are quite slow. It will improve dramatically once carbon-components-svelte ships support for v11 and related packages, `@carbon/styles`, https://github.com/carbon-design-system/carbon-components-svelte/issues/1629
